core-ui/src/components/Predefined/Details/Namespace/Namespace.details.js

```diff
@@ -63,11 +64,7 @@ export const NamespacesDetails = ({ DefaultRenderer, ...otherParams }) => {
     />
   ) : null;
 
-  const eventsParams = {
-    namespace: otherParams.resourceName,
-  };
-
-  const Events = <ComponentForList name="eventsList" params={eventsParams} />;
+  const Events = <EventsList namespace={otherParams.resourceName} />;
```

Comment:
maybe we could hide the `namespace` column when on a namespace details? it would give us more space for the remaining fields, that are all pretty wide

Reply:
Same thing in other places - do we need the `namespace` and `involved object` columns when being on the for example pods details page?

Reply:
I removed the namespace but left the involved object. 

Moreover, a namespace with the deep link has been added to the details view. The reason for it is that going from cluster-wide event list to details will redirect users to details in a specific namespace... since every event has a namespace. And we don't want them to overlook it